samcli/hook_packages/terraform/prepare/main.bat

```diff
@@ -0,0 +1,29 @@
+@echo off&setlocal
+:: redirect all stdout except for hook output to stderr
+(
+    :: ensure the script works regardless of where it's invoked from
+    for %%i in ("%~dp0.") do set "parent_path=%%~fi"
+    cd %parent_path%
+
+    :: check that python 3 is installed
+    py -3 --version >NUL 2>&1
+    if errorlevel 1 goto errorNoPython
+
+    :: set up virtual environment
+    py -3 -m venv ./venv
+    CALL venv/Scripts/activate
+
+    :: install dependencies
+    pip install -r requirements.txt
```

Comment:
this seems to assume the cwd of running this script is `/path/to/samcli/hook_packages/terraform/prepare/`. So back to the question about the location of the virtual env, are we creating the virtual env in the same cwd?

Reply:
addressed above